lib/action.js

```diff
@@ -24,19 +26,29 @@ class Action {
   }
 
   apply(issue, github) {
+    let args = {...this._action}
```

Comment:
```suggestion
    let action = {...this._action}
```

args feels unnecessarily generic